apps/manager_app/auth_card.c

```diff
@@ -111,6 +112,7 @@ typedef struct {
   char heading[30];
   char message[100];
   uint8_t acceptable_cards;
+  uint8_t family_id[FAMILY_ID_SIZE];
   bool pair_card_required;
 } auth_card_screen_ctx_t;
```

Comment:
Is this structure a duplicate of `card_sign_data_config_t` ?

Reply:
`card_sign_data_config_t` takes a pointer and the `auth_card_screen_ctx_t` object member is passed to that config.

This is helpful when FF value is passed in family id.